tests/distributions/test_lkj.py

```diff
@@ -18,7 +18,7 @@ def test_constraint(value_shape):
     value.diagonal(dim1=-2, dim2=-1).exp_()
     value = value / value.norm(2, dim=-1, keepdim=True)
 
-    assert (constraints.corr_cholesky_constraint.check(value) == 1).all()
+    assert (constraints.corr_cholesky.check(value) == 1).all()
```

Comment:
Likewise, this file can be removed now.

Reply:
Keeping file to test the `LKJ` distribution and backwards compatibility.